tests/integration_tests/build/test_coverage.py

```diff
@@ -17,9 +17,9 @@
 # Checkout the cpuid crate. In the future other
 # differences may appear.
 if utils.is_io_uring_supported():
-    COVERAGE_DICT = {"Intel": 77.81, "AMD": 78.54, "ARM": 72.26}
+    COVERAGE_DICT = {"Intel": 77.81, "AMD": 78.54, "ARM": 72.17}
 else:
-    COVERAGE_DICT = {"Intel": 75.53, "AMD": 76.26, "ARM": 69.58}
+    COVERAGE_DICT = {"Intel": 75.53, "AMD": 76.26, "ARM": 69.49}
```

Comment:
Is that because the newly added functions are not covered with tests? Is there a reason why we can't test them now?

Reply:
Added additional tests to avoid this.